coucharchive

```diff
@@ -282,6 +282,28 @@ def replicate_couchdb_server(source_url, target_url,
         pool.join()
 
 
+def bug_1418_create_missing_documents(source_db, target_db):
+    # Temporary function to be deleted when CouchDB team fixes the bug.
+    # It manually creates document on the target database that where
+    # previously deleted and re-created on the source.
+
+    missing_documents = [doc for doc in source_db if doc not in target_db]
+
+    for doc_id in missing_documents:
+        source_doc = source_db[doc_id]
+
+        # Not needed. Will be generated on target.
+        # Plus avoids conflicts.
+        del source_doc['_rev']
+
+        _, target_rev = target_db.save(source_doc)
```

Comment:
Shouldn’t we also copy by hand the `_security` document for documents on the `_users` database?

Reply:
What do you mean?

The `_security` document is a per-database doc, right? (There is only one per database?)
So, it should be copied here, no?
https://github.com/adrienverge/coucharchive/blob/836739b6b7c2192b3339f68e8e7c13fa033e748d/coucharchive#L318

Reply:
Oh yeah - I thought it was specific to documents in the `_users` database. Ignore my comment.